### ISSUE TITLE
[BUG] unable to load via Prusa connect Italian firmware 6.0.2 on mini

### ISSUE THREAD

**Issue by SNDR0** [prusa3d/Prusa-Firmware-Buddy#4035]
### Printer model

Mini 

### Firmware version

6.0.2

### Upgrades and modifications

Original

### Printing from...

Send firmware file downloaded from GitHub and from prusa

### Describe the bug

Only Italian firmware is refused by Prusa connect. Other languages works perfectly.

### How to reproduce

Send the Italian version for mini 6.0.2 via Prusa connect and an error message appears

### Expected behavior

Little bug while compiling..

**Comment by CZDanol** [comment]
Hey, what is the error message?

**Comment by CZDanol** [comment]
Internal ticket: BFW-5697

**Comment by SNDR0** [comment]
@CZDanol 
> Hey, what is the error message?

<img width="274" alt="Screenshot (2)" src="https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50074259/b7a3a561-5fe0-4666-acc4-53a4082bdbfb">
and if i scroll through the list of available firmware italian is missing
<img width="406" alt="Screenshot (1)" src="https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50074259/8cdb8dcf-061a-4bed-b204-f61fc77f7313">

**Comment by ClaGre** [comment]
That's right but I've upgraded the firmware in the usually way; download and place the firmware file into usb key.
Beware because before I've try to upload the file by Prusa Connect but without success because Prusa Connect say me that the file isn't correct.
